tests/v2/validation/provisioning/README.md

```diff
@@ -6,13 +6,25 @@ These are the examples of the configurations needed to run the provisioning pack
 ---
 
 ### Provisioning Input
-provisioningInput is needed to the run the RKE1 tests, specifically kubernetesVersion, cni, and providers. nodeProviders is only needed for custom cluster test; the framework only supports custom clusters through aws/ec2 instances.
+provisioningInput is needed to the run the RKE1 tests, specifically kubernetesVersion, cni, and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, node pools are divided by "{nodepool},". nodeProviders is only needed for node provider cluster tests.
 
 ```json
 "provisioningInput": {
-    "kubernetesVersion": ["v1.21.6+rke2r1"],
+    "nodesAndRoles": [
+      {
+        "etcd": true,
+        "controlplane": true,
+        "worker": true,
+        "quantity": 1,
+      },
+      {
+        "worker": true,
+        "quantity": 2,
+      }
+    ],
+    "kubernetesVersion": ["v1.24.2-rancher1-1"],
     "cni": ["calico"],
-    "providers": ["linode", "aws", "do", "harvester"],
+    "providers": ["linode", "aws", "azure"],
```

Comment:
harvester is still supported here, right?

Reply:
It should be right? Markus did you mean to delete this?

Reply:
Yeah, I had deleted it because currently, we support Linode, AWS, Azure. We don't officially support Harvester yet, so I had removed it, but I can put that back.